src/automata/CMakeLists.txt

```diff
@@ -1,4 +1,13 @@
 find_package(range-v3 REQUIRED)
+
 add_library(ta SHARED ta.cpp automata.cpp ata.cpp ta_regions.cpp)
 target_link_libraries(ta PRIVATE range-v3::range-v3 PUBLIC utilities)
 target_include_directories(ta PUBLIC include ${Boost_INCLUDE_DIR})
+
+find_package(Protobuf REQUIRED)
```

Comment:
One idea, based on experience: I think it might be beneficial to make all parser-related stuff optional.
Background: I experienced that, depending on the implementation, compilation takes a lot of time and in some cases, if you do not need the parser, this becomes annoying at some point.

Reply:
Yes, that makes total sense!

Reply:
Fixed in 61b6a9d94d1ce3f4644019898262620543231bfe